### ISSUE TITLE
flux:field.inline.fal crop problem after update 9.5.x -> 10.4.11

### ISSUE THREAD

**Issue by olegkarun** [FluidTYPO3/flux#1861]
I Use 
TYPO3 10.4.11
FLUX 9.4.2

Tryed GD and ImageMagic

The problem in cropvarians. For example

```
    <flux:field.inline.fal label='Background image' name="bg" required="1"  minItems="1"
                           enabledControls="{info:1,new:0,dragdrop:0,sort:0,hide:0}" 
                           allowedExtensions="jpg,jpeg,png" collapseAll="TRUE" 
                           cropVariants="{
                           
                                default: {
                                    title: 'Desktop',
                                    allowedAspectRatios: {
                                        default: { 
                                            title: '1920:780',
                                            value: '2.44'
                                        },
                                        NaN: {
                                         title: 'free',
                                         value: '0.0'
                                        }                          
                                    }           
                                },

                                mobile: {
                                    title: 'Mobile',
                                    selectedRatio: 'default',
                                    allowedAspectRatios: {
                                        default: {
                                            title: '320:350', 
                                            value: '0.914'
                                        },
                                        NaN: {
                                         title: 'free',
                                         value: '0.0'
                                        } 
                                    }
                                }           
                           }"/>

<f:image treatIdAsReference="1"
                         crop="{imageFal.crop}"
                         src="{imageFal.url}"
                         cropVariant="mobile" 
                         height="240"
                         />
```

When just added image and select crop area in editor all fine
![image](https://user-images.githubusercontent.com/3882942/102804758-dc14d000-43c2-11eb-8ecd-c2faf516bdaa.png)

But after save

![image](https://user-images.githubusercontent.com/3882942/102804815-f058cd00-43c2-11eb-9175-36a4994764aa.png)

Same problem when rendering in backend and frontend.

WIth TCA all works as aspected

**Comment by olegkarun** [comment]
Other problem found, may be connected with that. In front, if the image set just `with ` and no `height` - ratio is wrong like in issue. Later all worked perfectly.

**Comment by kimchristiansen** [comment]
TYPO3 10.4.11 had a image cropping bug. https://forge.typo3.org/issues/91855
The bug is solved in TYPO3 10.4.12
